content/kubernetes/deployment/quick-start.md

```diff
@@ -236,7 +236,7 @@ As part of the REC creation process, the operator stores the admission controlle
     kubectl patch ValidatingWebhookConfiguration redb-admission --patch "$(cat modified-webhook.yaml)"
     ```
 
-### Limit the webhook to the relevant namespaces
+### Limit the webhook to the relevant namespaces {#webhook}
```

Comment:
is this intended? I tried the link in the staging area and I am not sure where it leads.

Reply:
@yuvallevy2 this is an anchor that allows me to link to this specific section from other articles in our docs.